odl/tomo/backends/stir_bindings.py

```diff
@@ -170,7 +168,7 @@ def adjoint(self):
 
 class BackProjectorByBinWrapper(Operator):
 
-    """A backprojector using STIR."""
+    """A back projector using STIR."""
```

Comment:
I agree on "back projector". The concatenation of words is rather German thingy.